freshmaker/errata.py

```diff
@@ -116,14 +118,22 @@ def from_advisory_id(cls, errata, errata_id):
         security_impact = erratum_data["security_impact"].lower()
 
         has_hightouch_bug = False
+        has_compliance_priority_bug = False
         bugs = errata._get_bugs(erratum_data["id"]) or []
         for bug in bugs:
             if "flags" in bug and "hightouch+" in bug["flags"]:
                 has_hightouch_bug = True
+            if "flags" in bug and "compliance_priority+" in bug["flag"]:
+                has_compliance_priority_bug = True
+            if has_hightouch_bug and has_compliance_priority_bug:
                 break
+
         has_jira_major_incident = errata.has_jira_major_incidents(errata_id)
         is_major_incident = has_hightouch_bug or has_jira_major_incident
 
+        has_jira_compliance_priority = errata.has_jira_compliace_priority_label(errata_id)
```

Comment:
```suggestion
        has_compliance_priority_jira_issue = errata.has_compliance_priority_jira_issue(errata_id)
```
Minor name suggestion.

Reply:
That's a much clearer name! I'll change it in the function definition as well.